framework/dbt/basic/macros/overrides.sql

```diff
@@ -76,7 +47,7 @@
   ;
 {%- endmacro %}
 
-{% macro postgres__rename_relation(from_relation, to_relation) -%}
+{% macro cratedb__rename_relation(from_relation, to_relation) -%}
```

Comment:
Would not this override be something generic we would have as part of `dbt-cratedb2`?

Reply:
Yes, it should definitively go into dbt-cratedb2. Apparently, there is no software test for that in dbt-postgres, otherwise I believe it would have tripped on it?

Reply:
I am tracking a second round of improvements over here. That includes harvesting all others of the macros you provided.

- https://github.com/crate/dbt-cratedb2/issues/4

Reply:
Hi again. I've evaluated the situation around the `rename_relation` macro, and responded over here.

- https://github.com/crate/dbt-cratedb2/pull/2#pullrequestreview-2467215632